samples/anthropic/README.md

```diff
@@ -54,6 +54,9 @@ Send an HTTP POST request with your custom prompt:
 curl -i -X POST "${WEBHOOK_URL_POST}" -H "Content-Type: text/plain" -d "Why do cats purr?"
 ```
 
+- Replace `WEBHOOK_URL` with the URL of `on_http_post_with_prompt` webhook in the triggers section.
```

Comment:
you can now use https://autokitteh.readthedocs.io/en/latest/#autokitteh.get_webhook_url